source/codegen/proto_helpers.py

```diff
@@ -38,6 +38,8 @@ def get_grpc_type_from_ivi(type, is_array, driver_name_pascal):
     type = 'sint32'
   if 'ViInt64' in type:
     type = 'int64'
+  if 'ViUInt16' in type:
```

Comment:
I guess I was the first one to use ViUInt16 in our API.  I looked through the documentation for the proto standard, and it looks like there isn't a representation for 16 bit numbers.  Above this we are already turning ViInt8 into a 32 bit integer so I am doing the same here for the 16 bit type.